.semaphore/sandbox_deployment.yml

```diff
@@ -0,0 +1,26 @@
+version: v1.0
+name: Sandbox Deployment
+blocks:
+  - name: Deploy to Sandbox
+    task:
+      jobs:
+        - name: Deploy to Sandbox
+          commands:
+            - checkout
+            - sem-version ruby 2.5.1
+            - cache restore
+            - yarn install
+            - bundle install --deployment --path vendor/bundle
+            - cache store
+            - echo "Pretending to deploy to sandbox. Goodbye."
+            - bundle exec cap sandbox deploy
+      secrets:
+        - name: semaphore-deploy-key
+      prologue:
+        commands:
+          - chmod 600 ~/.ssh/semaphore_id_rsa
+          - ssh-add ~/.ssh/semaphore_id_rsa
```

Comment:
The `secrets` and `prologue` are where we load up our deploy key prior to our capistrano deployment. The SSH keypair can be managed here: https://resolvetosavelives.semaphoreci.com/secrets